consume-more-stuff/src/app/component/header/header.component.html

```diff
@@ -1,12 +1,12 @@
-<div>
-  <ul>
+<div class="header">
+  <ul class="header">
```

Comment:
Also don't want to use `<h2>` tags here, better to control font size/weight with CSS and not rely on non-semantic tags for stying.